fdk.go

```diff
@@ -129,10 +132,85 @@ type HTTPContext interface {
 	RequestMethod() string
 }
 
+// TracingContext contains all configuration for a function invocated to
+// get the tracing context data.
+type TracingContext interface {
+	/**
+	 * Returns true if tracing is enabled for this function invocation
+	 * @return whether tracing is enabled
+	 */
+	IsTracingEnabled() bool
+
+	/**
+	 * Returns the user-friendly name of the application associated with the
+	 * function; shorthand for Context.getAppName()
+	 * @return the user-friendly name of the application associated with the
+	 * function
+	 */
+	GetAppName() string
+
+	/**
+	 * Returns the user-friendly name of the function; shorthand for
+	 * Context.getFunctionName()
+	 * @return the user-friendly name of the function
+	 */
+	GetFunctionName() string
+
+	/**
+	 * Returns a standard constructed "service name" to be used in tracing
+	 * libraries to identify the function
+	 * @return a standard constructed "service name"
+	 */
+	GetServiceName() string
+
+	/**
+	 * Returns the URL to be used in tracing libraries as the destination for
+	 * the tracing data
+	 * @return a string containing the trace collector URL
+	 */
+	GetTraceCollectorURL() string
+
+	/**
+	 * Returns the current trace ID as extracted from Zipkin B3 headers if they
+	 * are present on the request
+	 * @return the trace ID as a string
+	 */
+	GetTraceId() string
+
+	/**
+	 * Returns the current span ID as extracted from Zipkin B3 headers if they
+	 * are present on the request
+	 * @return the span ID as a string
+	 */
+	GetSpanId() string
+
+	/**
+	 * Returns the parent span ID as extracted from Zipkin B3 headers if they
+	 * are present on the request
+	 * @return the parent span ID as a string
+	 */
+	GetParentSpanId() string
```

Comment:
small nit: in golang is not common to use the Get for the Get methond, can you please get rid of the Get prefix in these methods definitions?